### DISCUSSION TITLE
Lefavor soldering station compatibility

### DISCUSSION THREAD

**Discussion by bartosz-777** [deividAlfa/stm32_soldering_iron_controller#137] [Q&A]
Would this firmware work? Have anyone tried it?

It uses CKS32F and an square probably lcd w/b display
Here are pictures of mainboard
![image](https://github.com/user-attachments/assets/ef10cd82-e5d4-4bb0-aaf2-8fbc6c69c176)
![image](https://github.com/user-attachments/assets/8171e60a-087c-4b3b-ab24-4c9815222177)
![image](https://github.com/user-attachments/assets/78a76f8e-a268-4466-959b-ac2d17075822)
![image](https://github.com/user-attachments/assets/9c185876-7d34-43f8-add8-0ac3d74b3a4b)

**Comment by deividAlfa** [comment]
Will never work with such small mcu.
The gui is designed for 128x64 so another no.

**Reply by bartosz-777** [reply]
Ok, thanks for the quick response
